internal/pkg/template/templates/workloads/partials/cf/service-base-properties.yml

```diff
@@ -76,7 +76,10 @@ ServiceConnectConfiguration:
           {{- end}}
   {{- end}}
   {{- else}}
-  Enabled: False
+  !If
+    - IsGovCloud
+    - !Ref AWS::NoValue
+    - Enabled: False
```

Comment:
dumb question but instead of a condition with `!Ref AWS::NoValue` why not do:
```go
{{if .ServiceConnect}}
ServiceConnectConfiguration:

{{- end}}
```
This way we would get rid of an extra CFN condition?

Reply:
Why is it related to the gov-cloud 🤔 Like we need to determine if they are using gov-cloud or not.

Reply:
I think that's my point, wouldn't moving the if-conditional above the `ServiceConnectConfiguration` help not have to worry about any partitions?

For all partitions there would be the same behavior of No:Value if the user doesn't have `connect: true` in their manifest

Reply:
oooh but getting rid of `ServiceConnectConfiguration` wouldn't help you switch from enabling service connect to disabling is the reason behind, unless you explicitly specify `Enabled: False`.

Reply:
ooh what! that's surprising to me 🤔 my guess would have been that service connect gets disabled if the config is deleted

Reply:
Yeah that's how ecs update-service api works :)

Reply:
🤦 that makes sense